src/main/java/com/foo/gosucatcher/domain/estimate/presentation/ExpertResponseEstimateController.java

```diff
@@ -0,0 +1,68 @@
+package com.foo.gosucatcher.domain.estimate.presentation;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.foo.gosucatcher.domain.estimate.application.ExpertResponseEstimateService;
+import com.foo.gosucatcher.domain.estimate.application.dto.request.ExpertResponseEstimateCreateRequest;
+import com.foo.gosucatcher.domain.estimate.application.dto.request.ExpertResponseEstimateUpdateRequest;
+import com.foo.gosucatcher.domain.estimate.application.dto.response.ExpertResponseEstimateResponse;
+import com.foo.gosucatcher.domain.estimate.application.dto.response.ExpertResponseEstimatesResponse;
+
+import lombok.RequiredArgsConstructor;
+
+@RestController
+@RequestMapping("/api/v1/expert-response-estimates")
+@RequiredArgsConstructor
+public class ExpertResponseEstimateController {
+
+	private final ExpertResponseEstimateService expertResponseEstimateService;
+
+	@PostMapping("/{expertId}")
+	public ResponseEntity<ExpertResponseEstimateResponse> create(@PathVariable Long expertId,
+		@Validated @RequestBody ExpertResponseEstimateCreateRequest request) {
+
+		ExpertResponseEstimateResponse expertResponseEstimateResponse = expertResponseEstimateService.create(expertId,
+			request);
```

Comment:
```suggestion
		@Validated @RequestBody ExpertResponseEstimateCreateRequest request) {
		ExpertResponseEstimateResponse expertResponseEstimateResponse = expertResponseEstimateService.create(expertId,
			request);
```
이 부분은 개행 하지 않으셔도 될 것 같습니다~